lib/elaborate/elaborateNames.ts

```diff
@@ -26,6 +26,7 @@ export class ElaborateNames {
     }
   }
   public static async elaborate(vhdlLinter: VhdlLinter) {
+    // const start = Date.now();
```

Comment:
Maybe we should use some kind of debug utility and don't just comment out debug...
For now maybe remove all those debug statements?